charts/pgadmin4/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if and (.Values.serviceAccount.enabled) (not .Values.serviceAccount.name) }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+  {{- include "pgadmin.labels" . | nindent 4 }}
+  name: {{ include "pgadmin.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
If not mistaken ServiceAccount can also have annotations, would you be so kind to also add this?